### PR TITLE
Run s3 sync on AWS

### PR DESCRIPTION
**Pull request by RachaelBooth**

### CODE REVIEW

**Comment by RachaelBooth** [comment]
N.B. This is currently only wired up for staging, and the bucket names there are actually for review (for testing proof of concept). Those will need updating to actually use.

File: terraform/modules/s3_migration/kms.tf [hunk 1]
```diff
@@ -0,0 +1,29 @@
+resource "aws_kms_key" "this" {
+  description         = "KMS key used for the Secrets Manager secrets."
+  enable_key_rotation = true
+}
+
+resource "aws_kms_alias" "this" {
+  name          = "alias/${var.prefix}-db-migration-secretsmanager-secrets"
```
Comment (Chirag-Bhatti) [inline terraform/modules/s3_migration/kms.tf:7]: Should be `...s3-migration` rather than `...db-migration`

Comment (RachaelBooth) [reply terraform/modules/s3_migration/kms.tf:7]: Ooh, good point I missed that one.

**Review by SamSeed-Softwire — COMMENTED** [review]
Lgtm but I will avoid approving for now in case we run into any issues with it when doing the migration today.

File: terraform/staging/main.tf [hunk 2]
```diff
@@ -55,7 +55,9 @@ locals {
   database_port    = 5432
   redis_port       = 6379
 
+  # separate for testing
```
Comment (SamSeed-Softwire) [inline terraform/staging/main.tf:58]: Do we still want this comment?

File: terraform/staging/main.tf [hunk 3]
```diff
@@ -224,6 +226,30 @@ module "database_migration" {
...
+  source = "../modules/s3_migration"
+
+  count = local.create_s3_migration_infra ? 1 : 0
+
+  ecr_repository_url = "815624722760.dkr.ecr.eu-west-2.amazonaws.com/s3-migration"
+
+  prefix = local.prefix
+  buckets = {
+    export = {
+      source      = "s3://paas-s3-broker-prod-lon-791e541d-6996-44c8-9884-322448062d7b",
```
Comment (SamSeed-Softwire) [inline terraform/staging/main.tf:239]: I've updated these bucket urls to be correct for staging.

**Review by SamSeed-Softwire — APPROVED** [review]
Lgtm, only a couple of minor comments.

**Pull request merged**